scripts/update_api.py

```diff
@@ -73,7 +73,7 @@ def is_obj(ty):
 
 # Mapping to .NET types
 Type2Dotnet = { VOID : 'void', VOID_PTR : 'IntPtr', INT : 'int', UINT : 'uint', INT64 : 'Int64', UINT64 : 'UInt64', DOUBLE : 'double',
-                FLOAT : 'float', STRING : 'string', STRING_PTR : 'byte**', BOOL : 'byte', SYMBOL : 'IntPtr',
```

Comment:
This is very risky and could break the C# API. 
.Net is broken in release 4.7 or the one before because 
it used a non-working combination of 'bool' here and <stdbool.h>.